src/benchmark/file_based_benchmark.cpp

```diff
@@ -1,16 +1,13 @@
+#include <filesystem>
```

Comment:
One of the advantages of having our own cpplint was that we could add filesystem to the list of c++ includes. Having it at the top smells. Can we NOLINT this?

Reply:
The vanilla cpplint (now) lists filesystem there as well.